python/lsst/ts/watcher/rules/cp_verify_bias_alarm.py

```diff
@@ -0,0 +1,255 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CpVerifyBiasAlarm"]
+
+import yaml
+import json
+import lsst.daf.butler as dafButler
+
+from lsst.ts.idl.enums.Watcher import AlarmSeverity
+from lsst.ts import watcher
+
+
+class CpVerifyBiasAlarm(watcher.BaseRule):
+    """Report if cp_verify BIAS tests failed.
+
+    Set alarm severity NONE if the cp_verify tests passed,
+    and issue a WARNING if not.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+
+    Notes
+    -----
+    The alarm name is f"CpVerifyBiasAlarm.{name}:{index}",
+    where name and index are derived from ``config.name``.
+    """
+
+    def __init__(self, config):
+        remote_name = "OCPS"
+        # remote_index = 0: process events from both OCPS's
+        remote_index = 0
+        remote_info = watcher.RemoteInfo(
+            name=remote_name,
+            index=remote_index,
+            callback_names=["job_result"],
+            poll_names=[],
+        )
+        super().__init__(
+            config=config,
+            name=f"CpVerifyBiasAlarm.{remote_info.name}:{remote_info.index}",
+            remote_info_list=[remote_info],
+        )
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            description: Configuration for CpVerifyBiasAlarm
+            type: object
+            properties:
+                name:
+                    description: >-
+                        CSC name and index in the form `name` or `name:index`.
+                        The default index is 0.
+                    type: string
+                number_verification_tests_threshold_bias:
+                    type: integer
+                    descriptor: Minimum number of verification tests per detector per exposure per \
+                        test type that should pass to certify the bias combined calibration.
+                default: 8
+            required: [name]
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def __call__(self, topic_callback):
+        msg = topic_callback.get()
+        # OCPS response after calling the pipetask
+        response = json.loads(msg.result)
+        # boolean: did bias verification fail?
+        verify_bias_tests_pass = await self.get_verify_tests_pass_boolean(response)
+
+        if verify_bias_tests_pass:
+            return watcher.NoneNoReason
+        else:
+            # Tests did not pass. Follow up on images and calibrations is
+            # recommended.
+            return AlarmSeverity.SERIOUS, "FAULT state"
```

Comment:
In the docstring above you said it was supposed to be "WARNING" alarm but here you set it so SERIOUS.

Reply:
I adjusted the docstring to say ``SERIOUS``.

Reply:
I think we should WARNING instead